auth/src/main/java/org/jboss/pnc/bacon/auth/model/Credential.java

```diff
@@ -47,7 +47,7 @@ public boolean needsNewAccessToken() {
         if (!isValid()) {
             return true;
         } else {
-            return Instant.now().until(accessTokenExpiresIn, ChronoUnit.HOURS) < 20;
+            return Instant.now().until(accessTokenExpiresIn, ChronoUnit.HOURS) < 3;
```

Comment:
Since our access token will in 5 minutes in the future, this will probably always be true. I wonder if we handle this case and get new access token automatically while the PNC client object is already created but being used to submit requests for more than 5 minutes.